File: arrays_data_structure/arrays.py
import array

buffer = array.array('i', [1, 2, 3, 4, 5])
unit8array = array.array('B', [10, 50, 200, 120])
print(buffer)
print(f"Unit 8 Array {unit8array}")

unit8array[0] = 45
print(f"Unit 8 Array [0]:  {unit8array}")

unit8array[2] = 45
print(f"Unit 8 Array [2]:  {unit8array}")

File: arrays_data_structure/arrays.js
const a = new ArrayBuffer(6);
const a8 = new Uint8Array(a);
const a16 = new Uint16Array(a);
console.log(`Array buffer:`, a);

a8[0] = 45
console.log(`Unit 8 array: `, a8);

a16[0] = 0x4545
console.log(`Unit 16 array: `, a16)